modules/friendlytag.js

```diff
@@ -409,14 +463,42 @@ Twinkle.tag.updateSortOrder = function(e) {
 		return checkbox;
 	};
 
+	if(Twinkle.tag.canRemove) {
+		var generateAlreadyPresentTagsCheckboxes = function() {
```

Comment:
Technically it isn't since `var`s in JS have function scope rather than block scope, but still I guess this is bad practise.

Removed the condition. (the function needs to be available only in `canRemove` mode, but it doesn't for it be available either way)